soarca_fin_python_library/executor.py

```diff
@@ -172,14 +172,14 @@ def _handle_command_message(self, message: Command):
             try:
                 self._wait_for_ack(result.message_id)
                 break
-            except (Empty, TimeoutError) as e:
```

Comment:
Maybe log the error where you removed the e?

Reply:
Now calls log.debug. In the wait_for_ack method log.info and log.warning are called to provide some additonal context.